### PR TITLE
AWS NODE SETUP

### PR DESCRIPTION
**Pull request by alajko**
First Draft

### CODE REVIEW

File: aws-foundation-node/variables.tf [hunk 1]
```diff
@@ -0,0 +1,25 @@
+// EC2 instance configurations
+variable "node_instance_type" {
+  description = "Instance type of the node instance"
+  default     = "m5.large"
```
Comment (LeoHChen) [inline aws-foundation-node/variables.tf]: we use m5a.large by default

File: aws-foundation-node/sg.tf [hunk 2]
```diff
@@ -1,50 +1,8 @@
...
-  }
-}
 
 # Security Group with Harmoney Foundational Node ports enabled
 resource "aws_security_group" "foundational-sg" {
   name        = "Harmony Foundational Node Security Group"
   description = "Security group for Harmony Foundational Nodes"
-  vpc_id      = "${aws_vpc.vpc.id}"
 
   ingress {
```
Comment (LeoHChen) [inline aws-foundation-node/sg.tf:7]: enable 22 from everywhere is risky.

**Review by LeoHChen — COMMENTED** [review]
why the original folder was removed?  I think you should rename them and then make modification, so that we can easily track the original changes. Also, you should provide option for ppl to choose their own keys.

File: aws-foundation-node/variables.tf (hunk 1 above)
Comment (alajko) [reply aws-foundation-node/variables.tf]: Changed it.

File: aws-foundation-node/sg.tf (hunk 2 above)
Comment (alajko) [reply aws-foundation-node/sg.tf:7]: Reccomendation take from [here](https://docs.google.com/document/d/1t4TCcvqu84YK5657ddOrezJc_Dn8GRiIwGMwPGlk6x8/edit#)

**Comment by alajko** [comment]
> why the original folder was removed? I think you should rename them and then make modification, so that we can easily track the original changes. Also, you should provide option for ppl to choose their own keys.

Added option to choose Keys.

**Comment by alajko** [comment]
> why the original folder was removed? I think you should rename them and then make modification, so that we can easily track the original changes. Also, you should provide option for ppl to choose their own keys.

**Comment by alajko** [comment]
> why the original folder was removed? I think you should rename them and then make modification, so that we can easily track the original changes. Also, you should provide option for ppl to choose their own keys.

We need to make aws, specific folder so original one was removed.

**Pull request merged**